AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -41,70 +42,81 @@ @interface ASVideoNode ()
 
 @implementation ASVideoNode
 
+//TODO: Have a bash at supplying a preview image node for use with HLS videos as we can't have a priview with those
+
+
+#pragma mark - Construction and Layout
+
 - (instancetype)init
 {
-  if (!(self = [super init])) {
-    return nil;
-  }
-    
-  _previewQueue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0);
+	_previewQueue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0);
```

Comment:
@appleguy I couldn't find this one in the master branch, so I assume it's sorted